test/cluster/mv/test_mv_building.py

```diff
@@ -236,3 +236,100 @@ async def progress_saved():
+@skip_mode("release", "error injections are not supported in release mode")
+async def test_backoff_when_node_fails_task_rpc(manager: ManagerClient):
+    """
+    Scenario:
+    1. Set up a cluster. Two racks are sufficient.
+    2. Create the schema. Fill in the table with enough data so that the view
+       building won't finish immediately.
+    3. Create a view.
+    4. Enable the error injection that will simulate gossip taking time with marking
+       nodes as DOWN.
+    5. Stop the target node.
+    6. Wait for the view building coordinator to report that sending an RPC
+       to the target node has failed.
+    7. Disable the injection and revive the target node.
+    8. The total number of warnings should be small.
+    """
+
+    # We'll count messages in the logs, so let's make it easier.
+    cmdline = ["--smp=1"]
+    # Not needed, but it will be of tremendous help if we end up debugging a failure.
+    cmdline.extend(["--logger-log-level", "view_building_coordinator=trace",
+                    "--logger-log-level", "view_building_worker=trace",
+                    "--logger-log-level", "load_balancer=debug",
+                    "--logger-log-level", "storage_service=debug",
+                    "--logger-log-level", "raft_topology=debug"])
+
+    s1, s2 = await manager.servers_add(2, cmdline=cmdline, auto_rack_dc="dc1")
+
+    host_id1 = await manager.get_host_id(s1.server_id)
+    host_id2 = await manager.get_host_id(s2.server_id)
+
+    cql = manager.get_cql()
+
+    await cql.run_async("CREATE KEYSPACE ks WITH replication = {'class': 'NetworkTopologyStrategy', 'dc1': 2}")
+    await cql.run_async("CREATE TABLE ks.t (pk int, ck int, v int, PRIMARY KEY (pk, ck))")
+
+    row_count = 50_000
+    await asyncio.gather(*[cql.run_async(f"INSERT INTO ks.t (pk, ck, v) VALUES ({i}, {i + 1}, {i + 2})") for i in range(row_count)])
+
+    # Topology coordinator = view building coordinator.
+    topology_coordinator = await get_topology_coordinator(manager)
+    if topology_coordinator != host_id1:
+        s1, s2 = s2, s1
+        host_id1, host_id2 = host_id2, host_id1
+    logger.info(f"Coordinator: server ID={s1.server_id}, IP={s1.ip_addr}, host ID={host_id1}")
+    logger.info(f"Target: server ID={s2.server_id}, IP={s2.ip_addr}, host ID={host_id2}")
+
+    # Prevent the target node from processing everything too early.
+    await manager.api.enable_injection(s2.ip_addr, "view_building_worker_pause_build_range_task", one_shot=False)
+
+    log = await manager.server_open_log(s1.server_id)
+    mark = await log.mark()
+
+    # Make the view building coordinator ignore the gossiper. The purpose of this
+    # is to simulate a situation when the gossiper doesn't mark a dead node as
+    # such immediately. In a scenario like that, the view building coordinator
+    # will be retrying to send a request to it.
+    await manager.api.enable_injection(s1.ip_addr, "view_building_coorindator_ignore_gossiper", one_shot=False)
```

Comment:
Typo: `coorindator`->`coordinator`. The actual injection in the C++ code does not have this typo. Does this test work correctly? It does not trigger the injection.